src/scripting/lua_root.cc

```diff
@@ -360,10 +360,10 @@ void LuaEditor::__unpersist(lua_State* /* L */) {
  */
 
 /* RST
-World
------
+World and Tribe Descriptions
+----------------------------
 
-.. class:: World
+.. class:: Descriptions
 
    This offers access to the objects in the Widelands world and allows to add new objects.
    On how to build the world and adding new objects to it, see
```

Comment:
The line below this one (which GitHub won't let me comment on) links to "Scripts for World". It should additionally link to "Scripts for Tribes".